Makefile

```diff
@@ -97,6 +97,11 @@ all: lib xxhsum xxhsum_inlinedXXH
 ifeq ($(DISPATCH),1)
 xxhsum: CPPFLAGS += -DXXHSUM_DISPATCH=1
 xxhsum: xxh_x86dispatch.o
+DISPATCH_HEADERS = xxh_x86dispatch.h
+else ifeq ($(ARM64_DISPATCH),1)
```

Comment:
I rechecked the Makefile. Maybe it could be implemented by `uname -p` command. The command result is in below.
| Command | Mac on x86 | Mac on arm | Linux on x86 | Linux on arm |
|---------------| ---------------: | ---------------: | -----------------: | ----------------: |
| uname -p  | i386             | arm              | x86_64           | aarch64        | 

And I need to make sure that Makefile is only used in MacOS/Linux/UNIX in this project. CMAKE is used for Windows OS. Is it right? If so, I could create a macro to distinguish architecture.

Reply:
Sounds like a plan